text/0000-read-by-prefix.md

```diff
@@ -0,0 +1,111 @@
+- Feature Name: read_by_prefix
+- Start Date: 2018-12-19
+- RFC PR:
+- Sawtooth Issue:
+
+# Summary
+[summary]: #summary
+
+This RFC proposes an additional API to transaction context,
+context.listAddresses(address_prefix), which will get address prefix as input
+and return list of all existing addresses that starts with the prefix.
+
+This capability exists when reading state from rest-api but is missing when 
+transaction processor reads state.
```

Comment:
This is misleading, as it is not "missing" in the sense that we forgot to add it, but rather, it was excluded as uninteresting and not performant. Suggest "This capability exists in the Sawtooth REST API, but is not currently available via the transaction processor protocol or SDKs (for reasons covered in 'Drawbacks' below)."